apertium-ibo.ibo.lexc

```diff
@@ -242,24 +344,98 @@ ukwu:ukwu ADJ ; ! "big"
 obele:obele ADJ ; ! "small"
 niile:niile ADJ ; ! "all, each, every"
 dum:dum	ADJ ; ! "all, each, every"
+n'èzí:n'èzí ADJ ; ! "outside"
+n'azụ:n'azụ ADJ ; ! "behind"
+mma:mma ADJ ; ! "nice"
+nnukwu:nnukwu ADJ ; ! "big"
+obere:obere ADJ ; ! "small"
+ochie:ochie ADJ ; ! "old"
+nwatakịrị:nwatakịrị ADJ ; ! "little"
+oyi:oyi ADJ ; ! "cold"
+mechara:mechara ADJ ; ! "finished"
+nyocha:nyocha ADJ ; ! "searching"
+na-anwa:na-anwa ADJ ; ! "trying"
+obi-ụtọ:obi-ụtọ ; ! "happy"
+abia:abia ; ! "coming"
+ọkụ:ọkụ; ! "warm, hot"
+nke-ha:nke-ha ; ! "theirs"
+nso:nso ; ! "closer"
+
 
 !===================!
  LEXICON Prepositions
 !===================!
 
-na:na PREP ; ! "in, at, on"
+na:na PREP ; ! "in, at, on , with"
+site:site PREP ; ! "through, site" 
+n'azu:n'azu ; ! "behind"
+nke:nke ; ! "of"
+
+
+!=================!
+ LEXICON Adverbs
+!=================!
+
+Ebee:Ebee ADV ; ! "where"
+agaghị:agaghị ADV ; ! "never"
+mba:mba AdV ; ! "no"
+n'ihu:n'ihu  Adv ; ! "front"
+afọ:afọ ADV ; ! "years"
+nso:nso ADV ; ! "near"
+gburugburu:gburugburu ADV ; ! "around"
+eleghị-anya:eleghị-anya ADV ; ! "probably"
+nwayọọ-nwayọọ:nwayọọ-nwayọọ ADV ; ! "slowly"
+ngwa-ngwa:ngwa-ngwa ; ! "quickly"
+ka:ka ADV ; ! "still"
+n'okpuru:n'okpuru ; ! "under"
+ọ-bụghị:ọ-bụghị ; ! "not"
+mgbe-ahụ:mgbe-ahụ ; ! "then"
+mgbe:mgbe ; ! "when"
+ọ-bụ:ọ-bụ ; ! "it-is"
+ugbua:ugbua ; ! "now"
+ọzọ:ọzọ ; ! "again"
+nke-oma:nke-oma ; ! "well, carefully"
+ebeahu:ebeahu ; ! "there"
+nwere-ike:nwere-ike ; ! "could"
+n'ebe-nile:n'ebe-nile ; ! "everywhere"
+ụnyaahụ:ụnyaahụ ; ! "yesterday"
+ubochita:ubochita ; ! "today"
+mgbe-niile:mgbe-niile ; ! "always"
+ọnụ:ọnụ ; ! "together"
+ya-mere:ya-mere ; ! "so"
+maka-gini:maka-gini ; ! "why"
+
+
+
+!=================!
+ LEXICON Conjuction
+!=================!
+
+na:na CNJCOO ; ! "and"
+mana:mana CNJCOO ; ! "but"
+
+ọ-bụ-ezie-na:ọ-bụ-ezie-na CNJSUB ; ! "although"
+
+!=================!
+ LEXICON Determiner
+!=================!
 
+nke-m:nke-m ; ! "my"
```

Comment:
I believe - needs to be escaped, with `%`.

Reply:
`-` doesn't need to be escaped in general lexc lines, though it does need to be escaped in regular expressions and xfst files.